R/plot2.R

```diff
@@ -255,12 +258,18 @@ plot2.default = function(
     
   if (is.null(xlim)) xlim = range(x, na.rm = TRUE)
   if (is.null(ylim)) ylim = range(y, na.rm = TRUE)
-  
+
+  if (!is.null(ymin)) ylim[1] <- min(c(ylim, ymin))
```

Comment:
Minor, unnecessary point but can we switch these `<-` to `=` for internal consistency?

Reply:
https://github.com/grantmcdermott/plot2/pull/35/commits/a90b17362c8b122e8b8d437b370ede719f72f509